packages/expo-battery/src/ExpoBattery.ts

```diff
@@ -1,2 +1,2 @@
-import { NativeModulesProxy } from 'expo-modules-core';
-export default NativeModulesProxy.ExpoBattery;
+import { requireNativeModule } from 'expo-modules-core';
+export default requireNativeModule('ExpoBattery');
```

Comment:
may require https://github.com/expo/expo/pull/19482 to fully work correctly

Reply:
Hah, yeah, your PR reminded me to do that 😂 Thanks!